cobc/config.def

```diff
@@ -206,9 +206,10 @@ CB_CONFIG_SUPPORT (cb_comment_paragraphs, "comment-paragraphs",
 CB_CONFIG_SUPPORT (cb_control_division, "control-division",
 	_("CONTROL DIVISION"))
 
-CB_CONFIG_SUPPORT (cb_partial_replacing_with_literal,
-		   "partial-replacing-with-literal",
-		   _("partial replacing with literal"))
+CB_CONFIG_SUPPORT (cb_partial_replace_when_literal_src,	/* Enables literals as */
+		   "partial-replace-when-literal-src", /* replacement operands. */
+	_("apply partial replacing with literal source operand even when it replaces with spaces only;\n"
+	  "                        * \"skip\" prevents such replacements."))
```

Comment:
I'm fine with the shortened name. The comment is nice - please add that directly in the complete line before the entry.

```
/* Enables literals as replacement operands. */
CB_CONFIG_SUPPORT (cb_partial_replace_when_literal_src, "partial-replace-when-literal-src",
	_("apply partial replacing with literal source operand even when it replaces with spaces only;\n"
	  "                        * \"skip\" prevents such replacements"))
```


the trailing period here and on line 198 is wrong, please drop that